### PR TITLE
Add Team chat section

### PR DESCRIPTION
**Pull request by Mikaela**
## Description

Resolves: #1065 
Resolves: #999 

Preview: https://deploy-preview-1067--privacytools-io.netlify.com/software/teamchat/

### CODE REVIEW

File: _includes/nav.html [hunk 1]
```diff
@@ -47,6 +47,7 @@
             <a class="dropdown-item" href="/software/encryption-tools/"><i class="fas fa-lock fa-fw"></i> File Encryption</a>
             <a class="dropdown-item" href="/software/file-sharing/"><i class="fas fa-file-export fa-fw"></i> File Sharing</a>
             <a class="dropdown-item" href="/software/im/"><i class="fab fa-telegram-plane fa-fw"></i> Instant Messenger</a>
+            <a class="dropdown-item" href="/software/teamchat/"><i class="fas fa-business-time fa-fw"></i>Team chat</a>
```
Comment (Mikaela) [inline _includes/nav.html]: Any icon suggestions?

File: _includes/sections/teamchat.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
+<h1 id="teamchat" class="anchor"><a href="#teamchat"><i class="fas fa-link anchor-icon"></i></a> Private friendly Team chat applications</h1>
+
+<div class="alert alert-warning" role="alert">
+  <strong>If you are currently using an team chat like Slack you should pick an alternative here.</strong>
```
Comment (Mikaela) [inline _includes/sections/teamchat.html]: I forgot that Discord exists and typing this comment I also remember Microsoft Teams being a thing. I think the biggest concerns for FOSS projects/teams are Slack and Discord though and I am sure both have nice links.

File: _includes/sections/teamchat.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
+<h1 id="teamchat" class="anchor"><a href="#teamchat"><i class="fas fa-link anchor-icon"></i></a> Private friendly Team chat applications</h1>
```
Comment (Mikaela) [inline _includes/sections/teamchat.html]: This needs attention as I am not sure any team chat is designed with privacy in mind as I recall reading that the reason Slack hasn't implemented E2EE is due to letting "your boss read your PMs".

File: _includes/sections/teamchat.html [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+<div class="alert alert-warning" role="alert">
+  <strong>If you are currently using an team chat like Slack you should pick an alternative here.</strong>
+</div>
+
+{% include cardv2.html
+title="Rocket.chat"
+image="/assets/img/tools/rocket.chat.png"
+description="Rocket.chat is open source platform for team communication. It has optional federation and experimental E2EE. <a href=\"https://rocket.chat/docs/user-guides/end-to-end-encryption/\"><span class=\"badge badge-warning\" data-toggle=\"tooltip\" title=\"The documentation says this of E2EE: 'Note: This feature is currently in alpha. Its also not yet supported on mobile''\">Experimental <i class=\"far fa-question-circle\"></i></a></span>"
```
Comment (Mikaela) [inline _includes/sections/teamchat.html]: Federation is also experimental.

File: _includes/sections/teamchat.html [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+website="https://rocket.chat/"
+forum=""
```
Comment (Mikaela) [inline _includes/sections/teamchat.html]: TODO

File: _includes/sections/teamchat.html [hunk 6]
```diff
@@ -0,0 +1,40 @@
+<h1 id="teamchat" class="anchor"><a href="#teamchat"><i class="fas fa-link anchor-icon"></i></a> Private friendly Team chat applications</h1>
+
+<div class="alert alert-warning" role="alert">
+  <strong>If you are currently using an team chat like Slack you should pick an alternative here.</strong>
+</div>
+
+{% include cardv2.html
+title="Rocket.chat"
+image="/assets/img/tools/rocket.chat.png"
```
Comment (Mikaela) [inline _includes/sections/teamchat.html]: TODO

File: _includes/sections/teamchat.html [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+android=""
+ios=""
+mac=""
+windows=""
+linux=""
+%}
+
+{% include cardv2.html
+title="Keybase"
+image="/assets/img/tools/keybase.png"
```
Comment (Mikaela) [inline _includes/sections/teamchat.html]: TODO

File: _includes/sections/teamchat.html [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+description='Blah blah blah. <span class="badge badge-warning" data-toggle="tooltip" title="The server code is not open source">experimental <i class="far fa-question-circle"></i> (<a href="https://github.com/keybase/client/issues/6374">GitHub issue</a>)</span>'
```
Comment (Mikaela) [inline _includes/sections/teamchat.html]: Obviously TODO, I just wanted a preview build out and me working on this visible to everyone.

File: _includes/sections/teamchat.html [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+website="https://keybase.io/"
+forum=""
```
Comment (Mikaela) [inline _includes/sections/teamchat.html]: TODO, is it OK to use existing topics?

File: _includes/sections/teamchat.html [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+mac=""
+windows=""
+linux=""
+web=""
+%}
+
+<h3>Worth Mentioning</h3>
+
+<ul>
+  <li><a href="https://mattermost.com/">Mattermost</a> - <span class="badge badge-warning">No E2EE</span> A free and open-source Slack clone. </li>
```
Comment (Mikaela) [inline _includes/sections/teamchat.html]: #1066 has links on the requests for E2EE, but I am not sure it's a big priority for businesses.

File: pages/software/teamchat.html [hunk 11]
```diff
@@ -0,0 +1,8 @@
+---
+layout: page
+permalink: /software/teamchat/
+title: "Team chat"
+description: "Discover private ways to chat with your team without letting big businesses read your messages."
```
Comment (Mikaela) [inline pages/software/teamchat.html]: Better ideas wanted?

**Comment by Mikaela** [comment]
Big thanks to @ghbjklhv1 for showing me how to add sections in #1064

File: _includes/sections/teamchat.html (hunk 3 above)
Comment (Mikaela) [reply _includes/sections/teamchat.html]: * https://www.vice.com/en_us/article/kzj5xn/why-slack-doesnt-have-end-to-end-encryption-boss
    * https://www.vice.com/en_us/article/8q89ba/dear-slack-please-add-otr

File: _includes/sections/teamchat.html (hunk 2 above)
Comment (Mikaela) [reply _includes/sections/teamchat.html]: * https://www.theregister.co.uk/2016/03/17/no_slack_for_open_sourcers/
* https://drewdevault.com/2015/11/01/Please-stop-using-slack.html

for Discord I couldn't find more than https://discordapp.com/privacy

File: _includes/sections/teamchat.html (hunk 7 above)
Comment (Mikaela) [reply _includes/sections/teamchat.html]: Note to self: reread #740

File: _includes/sections/teamchat.html (hunk 3 above)
Comment (Mikaela) [reply _includes/sections/teamchat.html]: Typo.

File: _includes/sections/teamchat.html (hunk 2 above)
Comment (Mikaela) [reply _includes/sections/teamchat.html]: Question: who is our target audience? I guess I am going by assumption that it's people who run open source teams (community managers?) and people who care and are unhappy with Slack or Discord?

Keybase advertises themselves for everyone and one audience that is also on my mind are language learners whom I only know existing in IRC (bigger languages), Discord (a lot of languages) and Slack (one team I found via Duolingo and keep forgetting to login).

File: _includes/sections/teamchat.html (hunk 4 above)
Comment (Mikaela) [reply _includes/sections/teamchat.html]: Oh and someone I chatted with IM apps said that they would be a bit wary of Rocket.chat due to it not being designed to federate and that possibly leaving more room for mistakes.

File: _includes/sections/teamchat.html (hunk 2 above)
Comment (Mikaela) [reply _includes/sections/teamchat.html]: Richard Stallman offers us the following:

* https://stallman.org/discord.html
* https://stallman.org/slack.html
* https://stallman.org/microsoft.html - no mention of Microsoft Teams though

via Matrix

**Comment by Mikaela** [comment]
Icon sources:

* https://github.com/keybase/client/blob/master/media/icons/icon-keybase-logo-windows-64.png
* https://rocket.chat/press (Logos & Assets)

File: _includes/sections/teamchat.html (hunk 9 above)
Comment (Mikaela) [reply _includes/sections/teamchat.html]: The existing thread was https://forum.privacytools.io/t/what-do-you-think-about-keybase-io/920?u=mikaela and I didn't feel comfortable reusing it.

File: _includes/sections/teamchat.html (hunk 8 above)
Comment (Mikaela) [reply _includes/sections/teamchat.html]: Experimental --> closed source server? centralized?

And I think I am out of time and cannot think of this further today.

**Comment by Mikaela** [comment]
TODO: I have forgotten the source code file.

**Comment by Mikaela** [comment]
TODO: audits, Keybase https://github.com/privacytoolsIO/privacytools.io/issues/1071

**Comment by Mikaela** [comment]
> My go-to is generally Matrix. Although, I haven't used it RocketChat seems like a good alternative if it is truly FLOSS.

Currently it has too many issues to be listed and it's in process of getting delisted from the instant messengers page.

> I believe GNU also has some recommended packages, I'll see if I can find them.
Edit: Jami and GNU Mailman, although they aren't really Slack alternatives they are still useful.

We seem to list Jami as a VoIP app at https://www.privacytools.io/software/voip/ and GNU Mailman is a mailing list software (or am I confusing it to something similar?) and due to email being bad with privacy, I wouldn't include it.

**Comment by ghbjklhv1** [comment]
> email being bad with privacy

TBF it isn't much if any worse than many of the recommended instant messengers.
Email can now be encrypted fairly easily and has newer protocols like [Darkmail](https:/www.wikipedia.org/wiki/Darkmail).

Plus there is the Mixmaster anonymous remailer and whatnot:
https://www.wikipedia.org/wiki/Mixmaster_anonymous_remailer

From an ease of use perspective, it ain't great.
But it can be configured to work well.

File: _includes/sections/teamchat.html (hunk 2 above)
Comment (nitrohorse) [reply _includes/sections/teamchat.html]: >     * https://www.theregister.co.uk/2016/03/17/no_slack_for_open_sourcers/
> 
>     * https://drewdevault.com/2015/11/01/Please-stop-using-slack.html
> 
> 
> for Discord I couldn't find more than https://discordapp.com/privacy

[From Discord, 2 years ago](https://web.archive.org/web/20171029114027/https://feedback.discordapp.com/forums/326712-discord-dream-land/suggestions/17094256-implement-whispersystems-encryption-for-voice-and):

> E2E encryption is not a focus nor currently planned...

discord-e2ee

Comment (nitrohorse) [reply _includes/sections/teamchat.html]: > Question: who is our target audience? I guess I am going by assumption that it's people who run open source teams (community managers?) and people who care and are unhappy with Slack or Discord?

Great question -- I'd agree with your assumption :+1:

File: _includes/nav.html (hunk 1 above)
Comment (nitrohorse) [reply _includes/nav.html]: group-chat

How about [fa-comments](https://fontawesome.com/icons/comments?style=solid)?

```html
<i class="fas fa-comments"></i>
```

File: _includes/sections/teamchat.html (hunk 5 above)
Comment (nitrohorse) [reply _includes/sections/teamchat.html]: https://forums.rocket.chat/ ?

File: _includes/sections/teamchat.html [hunk 12]
```diff
@@ -0,0 +1,40 @@
...
+android=""
+ios=""
+mac=""
+windows=""
+linux=""
+%}
+
+{% include cardv2.html
+title="Keybase"
+image="/assets/img/tools/keybase.png"
```
Comment (nitrohorse) [inline _includes/sections/teamchat.html]: Found a higher quality image [here](https://commons.wikimedia.org/wiki/File:Keybase_logo_official.svg):

keybase1

File: pages/software/teamchat.html (hunk 11 above)
Comment (nitrohorse) [reply pages/software/teamchat.html]: Hmm... so here's the IM description:

> Discover secure and private ways to message others online without letting any third parties read your messages.

This is a bit boring but consistent?
> Discover secure and private ways to message your team without letting any third parties read your messages.

**Comment by Perelandra0x309** [comment]
> We seem to list Jami as a VOIP

https://git.ring.cx/savoirfairelinux/ring-project/wikis/features/All-features-by-client
According to this Jami does not have group chat. I don't use Jami so can't confirm right now.

File: _includes/nav.html (hunk 1 above)
Comment (JonahAragon) [reply _includes/nav.html]: I'd rather change the IM icon from `fa-telegram-plane` (which makes no sense as an icon) to `fa-comments` and use something like this for team chat:

image

```
<i class="fas fa-users fa-fw"></i>
```

File: _includes/sections/teamchat.html (hunk 8 above)
Comment (JonahAragon) [reply _includes/sections/teamchat.html]: `The server side of Keybase runs on proprietary code and is centralized`?

File: pages/software/teamchat.html (hunk 11 above)
Comment (JonahAragon) [reply pages/software/teamchat.html]: > Discover secure and private ways to communicate with your team online without letting any third parties read your messages.

File: _includes/sections/teamchat.html (hunk 12 above)
Comment (JonahAragon) [reply _includes/sections/teamchat.html]: FYI we don't use SVGs (for some security reason apparently, I think issues with Tor Browser?). So all our images are kinda poor quality on HiDPI displays :/

Comment (nitrohorse) [reply _includes/sections/teamchat.html]: Ah, gotcha - okay, we can probably than just convert it to a png then.

**Comment by JonahAragon** [comment]
Thinking about @five-c-d's https://github.com/privacytoolsIO/privacytools.io/issues/999#issuecomment-513569992, would it perhaps make more sense to list Team Chat as a separate section on the existing Instant Messengers page? Or merge IM, VoIP, and this category into one "Communication Tools" page (or something) with separate subheadings for each of them?

**Comment by nitrohorse** [comment]
@JonahAragon I can see having sub-sections under a "Communication Tools" umbrella page being overall more helpful and easier to navigate and compare/contrast. I like that idea.

**Comment by Mikaela** [comment]
Sounds good to me, that page would just be ` {% include sections/teamchat.html %}` for the separate categories?

File: _includes/sections/teamchat.html (hunk 5 above)
Comment (Mikaela) [reply _includes/sections/teamchat.html]: No, it's our forum thread https://forum.privacytools.io/t/discussion-rocket-chat/1223

File: _includes/sections/teamchat.html (hunk 8 above)
Comment (Mikaela) [reply _includes/sections/teamchat.html]: I applied your suggestion, but Blah blah blah needs to change before resolving this conversation.

File: _includes/sections/teamchat.html (hunk 12 above)
Comment (Mikaela) [reply _includes/sections/teamchat.html]: Done, I hope it looks fine for you

File: _includes/sections/teamchat.html (hunk 4 above)
Comment (Mikaela) [reply _includes/sections/teamchat.html]: > RocketChat stores the quasi-end2end keys, server-side. And does not implement perfect forward secrecy. More like protonmail's security-model than like signalapp. RocketChat/Rocket.Chat#9999 (comment) Basically, if your decrypt-passphrase is every compromised (either by a trojan or a compromised server... or just, by you logging in from a browser or OS that is vulnerable to the badguys) then the badguys get your entire messaging-history, and that of every team you are on, correct?

* https://github.com/privacytoolsIO/privacytools.io/issues/999#issuecomment-518303585

File: _includes/sections/teamchat.html (hunk 3 above)
Comment (Mikaela) [reply _includes/sections/teamchat.html]: Suggestion:

> Team chat applications where everything is end-to-end encrypted *or* under control of your team administrator.

File: _includes/sections/teamchat.html (hunk 2 above)
Comment (Mikaela) [reply _includes/sections/teamchat.html]: > If your team is currently using service like [Discord] or [Slack], you should pick an alternative here.

Links either: https://www.theregister.co.uk/2016/03/17/no_slack_for_open_sourcers/ or https://drewdevault.com/2015/11/01/Please-stop-using-slack.html (I kind of prefer this one)

Discord: https://web.archive.org/web/20171029114027/https://feedback.discordapp.com/forums/326712-discord-dream-land/suggestions/17094256-implement-whispersystems-encryption-for-voice-and (I prefer) or https://stallman.org/discord.html ?

File: _includes/sections/teamchat.html (hunk 10 above)
Comment (Mikaela) [reply _includes/sections/teamchat.html]: https://github.com/privacytoolsIO/privacytools.io/issues/1066#issuecomment-519922407 - Mattermost should be delisted.

File: _includes/nav.html [hunk 13]
```diff
@@ -47,6 +47,7 @@
             <a class="dropdown-item" href="/software/encryption-tools/"><i class="fas fa-lock fa-fw"></i> File Encryption</a>
             <a class="dropdown-item" href="/software/file-sharing/"><i class="fas fa-file-export fa-fw"></i> File Sharing</a>
             <a class="dropdown-item" href="/software/im/"><i class="fab fa-telegram-plane fa-fw"></i> Instant Messenger</a>
+            <a class="dropdown-item" href="/software/teamchat/"><i class="fas fa-users fa-fw"></i>Team chat</a>
```
Comment (nitrohorse) [inline _includes/nav.html]: Missing a space here: ```</i> Team```

File: _includes/sections/teamchat.html [hunk 14]
```diff
@@ -0,0 +1,44 @@
...
+description='Keybase provides a hosted team chat with end-to-end encryption. It has also been <a href=\"https://keybase.io/docs-assets/blog/NCC_Group_Keybase_KB2018_Public_Report_2019-02-27_v1.3.pdf\">indepedently audited (PDF)</a>. <span class="badge badge-warning" data-toggle="tooltip" title="The server side of Keybase runs on proprietary code and is centralized">experimental <i class="far fa-question-circle"></i> (<a href="https://github.com/keybase/client/issues/6374">GitHub issue</a>)</span>'
```
Comment (nitrohorse) [inline _includes/sections/teamchat.html]: Link is currently broken, just needs to be updated to
```html
<a href="https://keybase.io/docs-assets/blog/NCC_Group_Keybase_KB2018_Public_Report_2019-02-27_v1.3.pdf">indepedently audited (PDF)</a>
```

File: _includes/sections/teamchat.html [hunk 15]
```diff
@@ -0,0 +1,44 @@
+<h1 id="teamchat" class="anchor"><a href="#teamchat"><i class="fas fa-link anchor-icon"></i></a>Team chat applications where everything is end-to-end encrypted or under the control of your team administrator</h1>
+
+<div class="alert alert-warning" role="alert">
+        <strong>If your project is currently using a team chat like <a href="https://web.archive.org/web/20171029114027/https://feedback.discordapp.com/forums/326712-discord-dream-land/suggestions/17094256-implement-whispersystems-encryption-for-voice-and">Discord</a> or <a href="https://drewdevault.com/2015/11/01/Please-stop-using-slack.html">Slack</a> you should pick an alternative here.</strong>
```
Comment (nitrohorse) [inline _includes/sections/teamchat.html]: Small grammatical suggestion:
> If your project ~~is currently using~~ currently uses a team chat like...

File: _includes/sections/teamchat.html [hunk 16]
```diff
@@ -0,0 +1,44 @@
+<h1 id="teamchat" class="anchor"><a href="#teamchat"><i class="fas fa-link anchor-icon"></i></a>Team chat applications where everything is end-to-end encrypted or under the control of your team administrator</h1>
```
Comment (nitrohorse) [inline _includes/sections/teamchat.html]: Missing a space here: </i> Team

File: _includes/sections/teamchat.html (hunk 14 above)
Comment (Mikaela) [reply _includes/sections/teamchat.html]: Won't that break it due to being in the card? I removed the backslashes though.

Comment (Mikaela) [reply _includes/sections/teamchat.html]: Apparently not.

**Comment by Mikaela** [comment]
I realized that I had forgotten Wire Pro entirely, but I don't think it can be immediately boosted to recommended as I am under the impression that no one has experience with Wire Pro. I think it's also a good excuse to create the worth mentioning section.

File: _includes/sections/teamchat.html [hunk 17]
```diff
@@ -0,0 +1,41 @@
+<h1 id="teamchat" class="anchor"><a href="#teamchat"><i class="fas fa-link anchor-icon"></i></a> Team chat applications where everything is end-to-end encrypted or under the control of your team administrator</h1>
+
+<div class="alert alert-warning" role="alert">
+        <strong>If your project currently uses a team chat like <a href="https://web.archive.org/web/20171029114027/https://feedback.discordapp.com/forums/326712-discord-dream-land/suggestions/17094256-implement-whispersystems-encryption-for-voice-and">Discord</a> or <a href="https://drewdevault.com/2015/11/01/Please-stop-using-slack.html">Slack</a> you should pick an alternative here.</strong>
```
Comment (Mikaela) [inline _includes/sections/teamchat.html]: Potential alternative link: https://spyware.neocities.org/articles/discord.html ? However it's difficult to open with IPFS Companion for some reason.

**Pull request merged**